index.js

```diff
@@ -20,11 +20,12 @@ module.exports = (nextConfig = {}) => {
         webpack,
         buildId,
         dev,
-        config: { distDir = '.next', pwa = {}, pageExtensions = ['tsx', 'ts', 'jsx', 'js', 'mdx'], experimental = {} }
+        isServer,
+        dir,
+        config: { distDir = '.next', pageExtensions = ['tsx', 'ts', 'jsx', 'js', 'mdx'], experimental = {} },
       } = options
-
-      let basePath = options.config.basePath
-      if (!basePath) basePath = '/'
+      const basePath = options.config.basePath || '/'
+      const { pwa = {} } = nextConfig
```

Comment:
This is the real fix, the rest is a bit of code cleanup.

Reply:
Thank you!!! It works 🎉